packages/contracts-bridge/package.json

```diff
@@ -28,6 +28,7 @@
     "test:unit": "FOUNDRY_PROFILE=bridge-ci forge test",
     "test:unit:local": "FOUNDRY_PROFILE=bridge forge test",
     "test:static-analyze": "rm -rf node_modules && yarn link:node_modules && slither .",
+    "test:coverage": "cd ../.. && scripts/coverage-report.sh bridge",
```

Comment:
can this be invoked without `cd` e.g. `../../scripts/coverage-report.sh bridge`

if not, why not